composer.lock

```diff
@@ -124,28 +124,28 @@
         },
         {
             "name": "doctrine/inflector",
-            "version": "2.0.4",
```

Comment:
This lock file was probably added by accident to the commit and should be removed. Updating dependencies should probably happen in a dedicated commit and PR.

Reply:
oh, yes, weird. i dont recall making any composer changes

Reply:
it's fixed :)